mptt/static/mptt/draggable-admin.js

```diff
@@ -402,7 +402,11 @@ django.jQuery(function($){
                 expandOrCollapseNode(treeNode(storedNodes[i]));
             }
         } else {
-            collapseTree();
+            if (DraggableMPTTAdmin.expandTreeByDefault) {
+                expandTree();
```

Comment:
not a fan of `dont_collapse_tree_on_first_visit` - it's always better to minimise the number of negatives in var names to make them easier to understand.

Otherwise not concerned about collapse vs expand. This seems fine to me 👍